gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-bin.zip
```

Comment:
Consider using latest 5.* for now(due to `Android Studio`)

Reply:
`-bin` -> `-all`

Reply:
Done, now we use 5.6.4.